build.gradle

```diff
@@ -8,18 +8,80 @@ buildscript {
         classpath group: 'net.minecraftforge.gradle', name: 'ForgeGradle', version: '3.+', changing: true
     }
 }
+plugins {
+    id 'eclipse'
+    id 'idea'
+    id 'java'
+    id 'maven-publish'
+}
 apply plugin: 'net.minecraftforge.gradle'
-
-//Only edit below this line, the above code adds and enables the necessary things for Forge to be setup.
-apply plugin: 'eclipse'
-apply plugin: 'idea'
-
 version = "$version"
+def env = System.getenv()
+if (env.BUILD_NUMBER) {
```

Comment:
Do we require a build number?

Reply:
I don't think we'll ever export it, but if Dire chooses to do add it to the Environment, I thought I might as well put it here... Should I remove it?

Reply:
doesn't hurt anyone so don't matter :P